packages/mobile/src/screens/signon/FirstFollows.tsx

```diff
@@ -38,7 +38,6 @@ const FirstFollows = (props: FirstFollowsProps) => {
 
   const handleArtistsSelected = () => {
     dispatch(signOnActions.finishSignUp())
-    dispatch(signOnActions.followArtists(followArtists.selectedUserIds))
```

Comment:
What if we kept this here, but just changed the takeLatest like you did? I don't have the full picture but it seemed the issue was the saga not getting triggered, not an issue with dispatching followArtists?

Reply:
Just left another comment, we can't do `followArtists` until the account is created, fetched, and populated in the store.  That's what the saga in the other file intended to do (by also waiting on `fetchAccountSucceeded` but failed.

Reply:
Ah yeah makes sense, thanks